vars/util.groovy

```diff
@@ -204,4 +204,22 @@ GIT INFORMATION REPORT
     } else {
         println '[WARNING] The variable GIT_INFORMATION_REPORT does not exist'
     }
+}
+
+def getNextVersion(String version, String type, String suffix = 'SNAPSHOT') {
```

Comment:
in case the `type` is an String we should asserts there.
`assert ['major', 'minor', 'micro'].contains(type)`

Reply:
I didn't get you on this point. Can you please explain

Reply:
just add `assert ['major', 'minor', 'micro'].contains(type)` at the very first line of the method
https://docs.oracle.com/javase/7/docs/technotes/guides/language/assert.html